Dockerfile.QA

```diff
@@ -44,19 +44,34 @@ FROM ${BUILD_IMAGE} AS build
 ENV DEBIAN_FRONTEND=noninteractive
 
 # image_preprocess custom backend requires opencv
+# go client requires golang
 RUN apt-get update && apt-get install -y --no-install-recommends \
             libopencv-dev \
-            libopencv-core-dev
+            libopencv-core-dev \
+            golang-go \
+            protobuf-compiler
+
+apt-get update && apt-get install -y --no-install-recommends \
+            golang-go \
+            protobuf-compiler
 
 WORKDIR /workspace
 RUN cd builddir && \
     make -j16 test-util custom-backend
 
+# need protoc-gen-go to generate go specific gRPC modules
+RUN go get github.com/golang/protobuf/protoc-gen-go && \
+    go get google.golang.org/grpc
+
+ENV PATH "$PATH:$(go env GOPATH)/bin"
```

Comment:
By putting the env and the above install command here they are not going to be included in the final CI/devel container? Notice that this is a multipart docker file. I think you want to put this all around line 264 and the ENV should be set at the end with the other ENV

Reply:
I believe the apt-get should also be moved to the CI container. Moved that to the CI container section as well